tarentula/datashare_client.py

```diff
@@ -194,3 +194,25 @@ def temporary_project(self, source=DATASHARE_DEFAULT_PROJECT, delete=True):
             if delete and project is not None:
                 self.delete_index(project)
         return project
+
+    def metadata_fields(self, index=DATASHARE_DEFAULT_PROJECT, document_type='Document'):
+        url = urljoin(self.elasticsearch_host, index)
+        mapping = requests.get(url, cookies=self.cookies, headers=self.headers).json()
```

Comment:
`cookies` and `headers` here are not necessary, we used them only with Datashare (to pass API key/credentials)